astropy/coordinates/tests/test_matching.py

```diff
@@ -165,8 +159,7 @@ def test_matching_method():
     assert len(idx1) == len(d2d1) == len(d3d1) == 20
 
 
-@pytest.mark.skipif(not HAS_SCIPY or OLDER_SCIPY,
-                    reason="Requires scipy > 0.12.0 ")
+@pytest.mark.skipif(not HAS_SCIPY, reason="Requires scipy > 0.12.0 ")
```

Comment:
If you're still up for it, I'd do `reason="Requires scipy"`, since the version has now become irrelevant.

Reply:
Done. Thanks for review!